tests/support/mod.rs

```diff
@@ -22,9 +22,11 @@ extern crate tokio;
 extern crate tokio_connect;
 extern crate tokio_current_thread;
 pub extern crate tokio_io;
+pub extern crate tokio_rustls;
```

Comment:
I'm not sure if this needs to be `pub` --- it looks like it's only used within the `support` module and need not be exposed to the test modules that depend on `support`.
```suggestion
extern crate tokio_rustls;
```